docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -36,12 +36,73 @@ and gas used.
 `linea_estimateGas` is the recommended method for estimating gas on Linea. See our 
 [reference page](../../reference/api/linea-estimategas.mdx) for more information. 
 
-Linea also supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
-[`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice), and
-[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+Linea also supports:
+- [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas)
+- [`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice)
+- [`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+
+## Gas pricing
```

Comment:
Question. This content reads like we're populating the Ethereum networks `extraData` block field? Is this correct? Or is the `extraData` block field for Linea being populated.

Reply:
@Filter94 could you advise please? My assumption is that Linea has an `extraData` field identical to Ethereum. I could make it clearer in this passage that we're referring to a Linea `extraData`: 

> These values are inserted into each block's `extraData` field—32 bytes available in every Ethereum 
> block to store any data—and then used by the sequencer and Linea Besu nodes running the correct 
> plugins to expose `linea_estimateGas`, in both cases with the same logic.

Reply:
Let me go to the definitions to make sure we're on the same page.

By Ethereum I don't mean Ethereum Mainnet. I mean the protocol, many EVM chains are compatible with. At the end of the day, most of the EVM chains are just Ethereum forks with different depth of customization if any. Linea almost doesn't change the vanilla Ethereum Clique protocol at all, the way it is now. That's why most of the Ethereum clients are compatible with Linea without modification.

extraData is a block field that is present on most of the Ethereum chains, including Ethereum Mainnet and Linea. On Linea we're using this field to distribute the information about gas pricing.